### ISSUE TITLE
DRL for Dialogue Generation

### ISSUE THREAD

**Issue by nabihach** [jiweil/Neural-Dialogue-Generation#1]
Just curious ... will you be adding the code for your paper 'Deep Reinforcement Learning for Dialogue Generation' in this repository?

**Comment by jiweil** [comment]
Yep ! It's on my todo list. But will be out of the states until the end of Feb. Will resume updating the repo after then.

**Comment by MarkWuNLP** [comment]
Hi, when will you release the code for 'Deep Reinforcement Learning for Dialogue Generation'? Thank you in advanced.

**Comment by ydlu** [comment]
looking forward to seeing your code for  'Deep Reinforcement Learning for Dialogue Generation'
